Mathlib/CategoryTheory/Monoidal/Braided/Transport.lean

```diff
@@ -0,0 +1,66 @@
+/-
+Copyright (c) 2025 Dagur Asgeirsson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Dagur Asgeirsson
+-/
+import Mathlib.CategoryTheory.Monoidal.Braided.Basic
+import Mathlib.CategoryTheory.Monoidal.Transport
+
+/-!
+
+# Transport a symmetric monoidal structure along an equivalence of categories
+-/
+
+universe v₁ v₂ u₁ u₂
+
+open CategoryTheory Category Monoidal MonoidalCategory
+
+noncomputable section
+
+variable {C : Type u₁} [Category.{v₁} C]
+variable {D : Type u₂} [Category.{v₂} D]
+
+namespace CategoryTheory.Monoidal
+
+instance Transported.instBraidedCategory (e : C ≌ D) [MonoidalCategory C] [BraidedCategory C] :
+    BraidedCategory (Transported e) :=
+  braidedCategoryOfFullyFaithful (equivalenceTransported e).inverse
+
+local notation "e'" e => equivalenceTransported e
+
+instance (e : C ≌ D) [MonoidalCategory C] [BraidedCategory C] :
+    (e' e).inverse.Braided where
+  braided X Y := by
+    simp [Transported.instBraidedCategory, braidedCategoryOfFullyFaithful,
+      braidedCategoryOfFaithful]
+
+instance (e : C ≌ D) [MonoidalCategory C] [BraidedCategory C] :
+    ((e' e).functor ⋙ (e' e).inverse).LaxBraided :=
+  Functor.LaxBraided.ofNatIso _ _ (e' e).unitIso
+
+instance (e : C ≌ D) [MonoidalCategory C] [BraidedCategory C] :
+    ((e' e).functor ⋙ (e' e).inverse).Braided where
```

Comment:
This is problematic because once both `functor` and `inverse` are (lax)braided, the composition automatically inherits a (lax)braided instance. These should be made definitions instead.